back/lib/error_reporter.rb

```diff
@@ -0,0 +1,36 @@
+# Simplified (not compatible) version of Rails ErrorReporter
+# https://github.com/rails/rails/pull/43625/files
+class ErrorReporter
+  class << self
+    # Report any unhandled exception, and swallow it.
+    #
+    #  ErrorReporter.handle do
+    #    1 + '1'
+    #  end
+    #
+    def handle(error_class = StandardError)
```

Comment:
What do you think about adding a test for the newly added method?

Reply:
Good point. Will do :+1: